src/pages/specification/specification-respec.html

```diff
@@ -19,6 +12,7 @@
       // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
       var respecConfig = {
         edDraftURI: 'https://webmonetization.org/specification',
+        latestVersion: 'https://webmonetization.org/specification',
```

Comment:
If we don't provide, default is `https://www.w3.org/web-monetization/` which is 404/.